messaging/Services/ConvertToIJEBackgroundWork.cs

```diff
@@ -31,13 +32,27 @@ public Message(long messageId)
 
       public class Worker : IBackgroundWorker<Message, Worker>
       {
-          private readonly ApplicationDbContext _context;
+        private readonly ApplicationDbContext _context;
 
-          public Worker(ApplicationDbContext context)
-          {
-              // This is where you put your dependencies, services, etc.
-              this._context = context;
-          }
+        public Worker(ApplicationDbContext context)
+        {
+            // This is where you put your dependencies, services, etc.
+            this._context = context;
+        }
+
+        private readonly static Dictionary<string, Action<CommonMessage, IncomingMessageItem, ApplicationDbContext>> CreateAckMessageCommand = new()
+        {
```

Comment:
Ah I see here you are creating the message acks for specific types. Can you add a summary comment to this to explain how you are using this? It's a bit hard to follow.